server/events/command_runner.go

```diff
@@ -179,7 +179,11 @@ func (c *DefaultCommandRunner) RunAutoplanCommand(logger logging.SimpleLogging,
 	err = c.PreWorkflowHooksCommandRunner.RunPreHooks(ctx)
 
 	if err != nil {
-		ctx.Log.Errorf("Error running pre-workflow hooks %s. Proceeding with %s command.", err, command.Plan)
+		ctx.Log.Errorf("Error running pre-workflow hooks %s. Aborting %s command.", err, command.Plan.String())
+		if err := c.VCSClient.CreateComment(ctx.Pull.BaseRepo, ctx.Pull.Num, fmt.Sprintf("Encountered an error during pre-workflow-hook execution: %s", err), command.Plan.String()); err != nil {
+			ctx.Log.Errorf("unable to comment: %s", err)
+		}
```

Comment:
I'd like to move away from this paradigm of commenting logging errors on the PR since its just noise that the customer won't understand and just pollutes our codebase all over the place.

The ideal UX in my mind is to mark the status/checks as fail and have some details as to why they failed when you click into it.  For now i think just failing the statuses is enough since customers will need our help anyways at that point.